documentation.html

```diff
@@ -392,12 +409,56 @@ <h4 class="display-inlineBlock">Toggle Button as Radio Button</h4>
               </div>
             </div>
           </section>
-          
+
+        </section>
+
+
+
+        <!-- Dark button class starts here -->
+        
+          <section class="instruction-block">
+            <h2 id="base-icon-btn">Dark-Mode Button</h2>
+            <div class="text">
+              The <code>dark-mode</code> class enables you to have a dark mode for a button by adding
+              <code>dark-mode</code> class to the button
+              or by adding <code>data-theme="dark"</code> to a container element like the body element.
+            </div>
+
+          <section class="instruction-block" id="dark-mode">
```

Comment:
Change the id here based on the id that you used for the link in the menu

Reply:
done